src/components/app-post/app-post.tsx

```diff
@@ -0,0 +1,52 @@
+import { Component, h, Prop, State } from '@stencil/core';
+import { RouterHistory } from '@stencil/router';
+import { BlogPostService, BlogPost } from '../../services/blogPost';
+
+@Component({
+  tag: 'app-post',
+  styleUrl: 'app-post.scss',
+  shadow: true,
+})
+export class AppPost {
+  @State() blogPostArr: BlogPost[];
```

Comment:
variable name blogPosts can indicate collection of blogPost instead of blogPostArr